homeassistant/components/amcrest/__init__.py

```diff
@@ -117,7 +122,15 @@ def _has_unique_names(devices):
 class AmcrestChecker(ApiWrapper):
     """amcrest.ApiWrapper wrapper for catching errors."""
 
-    def __init__(self, hass, name, host, port, user, password):
+    def __init__(
+        self,
+        hass: HomeAssistantType,
```

Comment:
Change `HomeAssistantType` to `HomeAssistant`.
https://github.com/home-assistant/core/blob/71b0f6d095603e2c574a46763765f079d367dc2e/homeassistant/helpers/typing.py#L28-L33

Reply:
Ah, that's good to know, removed the `HomeAssistantType` and `ServiceCallType` uses throughout.